dao/src/main/java/org/thingsboard/server/dao/sql/query/EntityKeyMapping.java

```diff
@@ -560,8 +560,23 @@ private String buildStringPredicateQuery(QueryContext ctx, String field, StringF
                 value = "%" + value + "%";
                 stringOperationQuery = String.format("%s not like :%s or %s is null)", operationField, paramName, operationField);
                 break;
+            case IN:
+                value = value.replaceAll("'","").replaceAll("\"", "");
```

Comment:
Move 
```java
value.replaceAll("'","").replaceAll("\"", "")
```
to a method with a meaningful name. Also add 
```java
List.of(value.trim().split("\\s*,\\s*"))
```
there to avoid extra switch on line 572